### PR TITLE
Add lazy loading to funders image.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by olasitarska** [comment]
Tested it locally, it works. @timgraham, wanna take a look?

File: djangoproject/static/js/main.js [hunk 1]
```diff
@@ -66,6 +65,11 @@ define(function() {
 		mods.push('mod/fundraising-donation');
 	}
 
+	//fundraising heros list
+	if (hasClass('heros-section')) {
+		mods.push('mod/fundraising-heros')
```
Comment (timgraham) [inline djangoproject/static/js/main.js]: missing semi-colon

File: djangoproject/static/js/mod/fundraising-heros.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+define(['jquery', 'jquery.unveil'], function ($){
+	$('.hero-logo img').unveil(200);  // 170 = size of image
```
Comment (timgraham) [inline djangoproject/static/js/mod/fundraising-heros.js]: could we add a comment explaining the purpose?

**Comment by timgraham** [comment]
Do we need the example images and copy of jquery?

File: djangoproject/static/js/mod/fundraising-heros.js (hunk 2 above)
Comment (jezdez) [inline djangoproject/static/js/mod/fundraising-heros.js]: sure

File: djangoproject/static/js/main.js (hunk 1 above)
Comment (jezdez) [inline djangoproject/static/js/main.js]: good catch

**Comment by jezdez** [comment]
@timgraham I don't think we need them, so I'll add them to the gitignore file. I rather let bower do its thing and not have to delete parts of the source it installs on the worry that it may prevent bower to work correctly

File: djangoproject/static/js/mod/fundraising-heros.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+define([
+	'jquery', //requires jquery
+	'jquery.unveil' // requires unveil.js
+	], function( $ ) {
```
Comment (timgraham) [inline djangoproject/static/js/mod/fundraising-heros.js]: no spaces around $?

File: djangoproject/static/js/mod/fundraising-heros.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+		// assumes an image size of 170 px plus some margin
+		var threshold = 200;
+
+		var FundraisingHeros = function(images) {
+			this.images = $(images); // the heros images
+			this.init();
+		};
+
+		FundraisingHeros.prototype = {
+			init: function(){
```
Comment (timgraham) [inline djangoproject/static/js/mod/fundraising-heros.js]: space before {

File: djangoproject/static/js/mod/fundraising-heros.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+				var self = this;
+				$(document).ready(function() {
+					self.images.unveil(threshold);
+				});
+			},
+
```
Comment (timgraham) [inline djangoproject/static/js/mod/fundraising-heros.js]: missing a closing parenthesis (file currently throws SyntaxError)

**Comment by timgraham** [comment]
Merged in 47e5cbbe7f204680917fd0bf2582b140e37fe2bb, thanks Jannis!

**Pull request closed without merging**